editor/editor_plugin.cpp

```diff
@@ -100,6 +100,11 @@ void EditorPlugin::remove_control_from_bottom_panel(Control *p_control) {
 	EditorNode::get_bottom_panel()->remove_item(p_control);
 }
 
+void EditorPlugin::set_dock_tab_icon(Control *p_control, const Ref<Texture2D> &p_icon) {
+	ERR_FAIL_NULL(p_control);
```

Comment:
EditorDockManager already does this check.

Reply:
It's the same as other methods. For example `add_dock()`:

https://github.com/godotengine/godot/blob/11d3768132582d192b8464769f26b493ae822321/editor/editor_plugin.cpp#L88-L91

https://github.com/godotengine/godot/blob/11d3768132582d192b8464769f26b493ae822321/editor/editor_dock_manager.cpp#L669-L671